src/scheduler/workers/analytics/googleClient.ts

```diff
@@ -52,6 +52,6 @@ export async function updateSheet(view: string, values: any[], valueInputOption
 
     return res;
   } catch (e) {
-    logger.error(`sending data for ${view} failed with error ${e}`);
+    logger.error(`updateSheet: sending data for ${view} failed with error ${e}`);
```

Comment:
this has it's own tagged logger, but you can leave as is 
https://github.com/UKForeignOffice/lists/pull/653/files#r1252172159

Reply:
ah I didn't notice that